scheduler/src/common/helper/garms.helper.ts

```diff
@@ -196,6 +212,125 @@ export const createGarmsCashFileDetails = (
   }
 };
 
+export const createGarmsCreditFile = (
+  transactions: Transaction[],
+  garmsExtractType: GarmsExtractType,
+  permitServiceCodes: Map<string, number>,
+  logger: Logger,
+) => {
+  const fileName = 'GARMS_CREDIT' + Date.now();
+  const logStream: fs.WriteStream = fs.createWriteStream(GARMS_LOCAL_FILE_PATH + fileName, {
+    flags: 'a',
+  });
+  try {
+    let serviceCount = 0;
+    let totalAmount = 0;
+    const date = new Date();
+    const groupedTransactionsByDate: DateTransaction[] =
+      groupTransactionsByDate(transactions);
+    if (groupTransactionsByDate && groupedTransactionsByDate.length > 0) {
```

Comment:
You meant this perhaps?
```suggestion
    if (groupedTransactionsByDate && groupedTransactionsByDate.length > 0) {
```

Reply:
that correct, thanks for pointing.